content/git/concepts/config/config.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Git Config
```

Comment:
This metadata section is incorrect please see look at another similar term entry or a template like the example linked - 
https://github.com/Codecademy/docs/blob/main/documentation/term-entry-template.md?plain=1